provider/ansible.py

```diff
@@ -132,17 +132,18 @@ def python_install():
             LOG_JOB.error("Failed to get available pip binary")
             return False
         install_cmd = '%s install ansible' % pip_bin
+        # Install ansible depended packages that can't be installed
+        # automatically by pip when installing ansible
+        package_list = params.objects('package_list')
```

Comment:
Need to set the default value to `sshapss`.

@yanan-fu, would you also help to review this one?

Reply:
Updated. Thanks.